go.mod

```diff
@@ -51,7 +51,7 @@ require (
 	github.com/google/wire v0.5.0
 	github.com/gorilla/websocket v1.4.2
 	github.com/gosimple/slug v1.12.0
-	github.com/grafana/cuetsy v0.0.3
+	github.com/grafana/cuetsy v0.0.4-0.20220714174355-ebd987fdab27
```

Comment:
just because I haven't rolled an actual release of it yet with the new version. (I need to get more disciplined about all this, now that we're finally ramping up)